stream-chat-android-ui-components/src/main/res/layout/stream_search_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="36dp"
+    android:background="@drawable/stream_shape_search_view_background"
+    >
+
+    <ImageView
+        android:id="@+id/searchIcon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="8dp"
+        android:contentDescription="@string/stream_search_view_search_icon_description"
+        android:src="@drawable/stream_ic_search"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+
+    <EditText
+        android:id="@+id/inputField"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
```

Comment:
I would also add a very small end margin here

Reply:
Good idea, added an `8dp` margin to be a bit more symmetrical

![image](https://user-images.githubusercontent.com/12054216/98917925-489bd580-24cd-11eb-8672-e4dc507f7131.png)